genie-client/src/test/java/com/netflix/genie/client/GenieClientsIntegrationTestsBase.java

```diff
@@ -84,12 +84,17 @@ Cluster constructClusterDTO(final String id) {
         configList.add("config1");
         configList.add("configs2");
 
+        final Set<String> dependenciesList = new HashSet<>();
```

Comment:
Just a FYI but you can use the Guava `Sets.newHashSet(item1, item2, ...)` method to shortcut this kind of thing